packages/icons/resources/icons/icons.json

```diff
@@ -3412,5 +3412,11 @@
     "iconName": "inherited-group",
     "tags": "group, inheritance, inherited, membership, people",
     "group": "interface"
+  },
+  {
+    "displayName": "Hat",
+    "iconName": "hat",
+    "tags": "head, clothing, cap",
```

Comment:
Can we add "role" which is the intended use (inside Palantir)? "Fedora" might make sense too.